app/src/main/java/io/numbersprotocol/starlingcapture/source/canon/CanonCameraControlService.kt

```diff
@@ -21,21 +26,39 @@ class CanonCameraControlService : Service(), CoroutineScope by CoroutineScope(Di
     private val notificationUtil: NotificationUtil by inject()
     private val foregroundNotificationId = notificationUtil.createNotificationId()
     private val errorNotificationId = notificationUtil.createNotificationId()
+    private lateinit var address: String
     private lateinit var canonCameraControlApi: CanonCameraControlApi
     private val proofCollector: ProofCollector by inject()
     private lateinit var foregroundNotificationBuilder: NotificationCompat.Builder
     private val beforeDestroyCallbacks = mutableSetOf<() -> Unit>()
 
     @FlowPreview
     override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
+//        bindProcessToWiFiNetwork()
```

Comment:
This expectedly breaks Internet traffic. Proper fix will come from https://github.com/numbersprotocol/starling-capture/issues/22